src/shared/utils/git-utils.tsx

```diff
@@ -32,7 +32,7 @@ export const detectGitType = (url: string): GitProvider => {
   if (hasDomain(url, 'bitbucket.org')) {
     return GitProvider.BITBUCKET;
   }
-  if (hasDomain(url, 'gitlab.com')) {
+  if (hasDomain(url, 'gitlab.com') || hasDomain(url, 'gitlab.cee.redhat.com')) {
```

Comment:
Why internal redhat gitlab is being hardcoded in upstream codebase?

This should be a config if you want to have custom mapping

Reply:
Good point. You're right, the hardcode is a bit weird.
And I guess selecting git provider directly is much cost-effective than developing more other extra custom mapping here.
So I would confirm with the customer to close the PR directly.